cmd/infracost/testdata/comment_bitbucket_help/comment_bitbucket_help.golden

```diff
@@ -0,0 +1,32 @@
+Post an Infracost comment to Bitbucket
+
+USAGE
+  infracost comment bitbucket [flags]
+
+EXAMPLES
+  Update comment on a pull request:
+
+      infracost comment bitbucket --repo my-org/my-repo --pull-request 3 --path infracost.json --bitbucket-token $BITBUCKET_TOKEN
+
+  Post a new comment to a commit:
+
+      infracost comment bitbucket --repo my-org/my-repo --commit 2ca7182 --path infracost.json --behavior delete-and-new --bitbucket-token $BITBUCKET_TOKEN
+
+FLAGS
+      --behavior string               Behavior when posting comment, one of:
+                                        update (default)  Update latest comment
+                                        new               Create a new comment
+                                        delete-and-new    Delete previous matching comments and create a new comment (default "update")
+      --bitbucket-server-url string   Bitbucket Server URL
+      --bitbucket-token string        Bitbucket access token
+      --commit string                 Commit SHA to post comment on, mutually exclusive with pull-request. Not available when bitbucket-server-url is set
+      --dry-run                       Generate comment without actually posting to Azure Repos
+  -h, --help                          help for bitbucket
+  -p, --path stringArray              Path to Infracost JSON files, glob patterns need quotes
+      --pull-request int              Pull request number to post comment on
+      --repo string                   Repository in format owner/repo
+      --tag string                    Customize text used to detect comments posted by Infracost. It is placed at the bottom of a comment
```

Comment:
@alikhajeh1 Please review the copy, specifically `--commit` and `--tag` ones. Should we also clarify anyhow the `--bitbucket-token` that it needs `user:password` combination for Cloud and HTTP Access Token for Server? We'll explain that in documentation too.

Reply:
Done.